mappings/net/minecraft/client/render/BufferBuilder.mapping

```diff
@@ -8,62 +8,65 @@ CLASS net/minecraft/class_287 net/minecraft/client/render/BufferBuilder
 	FIELD field_1565 format Lnet/minecraft/class_293;
 	FIELD field_1567 drawMode Lnet/minecraft/class_293$class_5596;
 	FIELD field_20884 elementOffset I
-	FIELD field_21594 textured Z
+	FIELD field_21594 canSkipElementChecks Z
+		COMMENT Whether this builder is aware of the vertex format and can skip checks
+		COMMENT for the current target element while building a vertex in {@link
+		COMMENT #vertex(float, float, float, float, float, float, float, float, float, int, int, float, float, float)}.
 	FIELD field_21595 hasOverlay Z
 	FIELD field_27348 sortingPrimitiveCenters [Lnet/minecraft/class_1160;
 	FIELD field_27349 sortingCameraX F
 	FIELD field_27350 sortingCameraY F
 	FIELD field_27351 sortingCameraZ F
-	FIELD field_27352 hasNoVertexBuffer Z
-	FIELD field_32050 MAX_BUFFER_SIZE I
+	FIELD field_27352 indexOnly Z
+	FIELD field_32050 BUFFER_ROUND_SIZE I
 	METHOD <init> (I)V
 		ARG 1 initialCapacity
-	METHOD method_1324 restoreState (Lnet/minecraft/class_287$class_5594;)V
+	METHOD method_1324 restoreSortState (Lnet/minecraft/class_287$class_5594;)V
```

Comment:
```suggestion
	METHOD method_1324 beginSortedElementBuffer (Lnet/minecraft/class_287$class_5594;)V
```
I think restoreSortState doesn't communicate that this actually deletes the old contents of the buffer